app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,38 @@
+!!! XML
```

Comment:
It it used when you want to use haml to render xml formats. But for now we don't need it, so I remove it.